libyaml-cpp: 0.5.3 -> 0.6.1

andir opened the pull request:
###### Motivation for this change

Bump to latest upstream version.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [X] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

andir committed:
libyaml-cpp: 0.5.3 -> 0.6.1

andir committed:
interception-tools: removed custom libyamlcppWithoutBoost

Since upstream libyaml-cpp does not reuqire boost anymore it is safe to
remove.

andir commented:
Trying to build a few of it's users:

@GrahamcOfBorg build interception-tools powerdns mongodb

GrahamcOfBorg reviewed (commented):
Failure on x86_64-darwin [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34426)

<details><summary>Partial log (click to expand)</summary><p>

```
Package ‘interception-tools-0.1.1’ in /Users/graham/nix-borg/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/grahamc-ndnd/pkgs/tools/inputmethods/interception-tools/default.nix:27 is not supported on ‘x86_64-darwin’, refusing to evaluate.

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.

```
</p></details>

GrahamcOfBorg reviewed (commented):
Failure on aarch64-linux [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34426)

<details><summary>Partial log (click to expand)</summary><p>

```
gcc -o build/nixos/third_party/wiredtiger/src/cursor/cur_index.o -c -std=c11 -I/nix/store/60q6d1a7s464izwbxq9gi0q5m3x6kjiw-cyrus-sasl-2.1.26-bin/include -I/nix/store/ybgzdhff90g8hrkv4c9863ynh9h4ldz6-boost-1.60.0/include -I/nix/store/5v9q166fyigywngdajcgh1wd8f0m62l5-gperftools-2.6.1/include -I/nix/store/hhz1a55j3mmwz6z4bc8crhm3qi5gm2l1-pcre-cpp-8.41-bin/include -I/nix/store/k2mcv7x6v4wrq9ldjzqa4by3g2fkk0zp-snappy-1.1.7/include -I/nix/store/wk9ngxwd8i3fiqhp4df201bqklfp2f4j-zlib-1.2.11/include -I/nix/store/rk8833kc9imj5mf4y1v3v6ykj4asfp18-libyaml-cpp-0.6.0/include -I/nix/store/60q6d1a7s464izwbxq9gi0q5m3x6kjiw-cyrus-sasl-2.1.26-bin/include -I/nix/store/sk5fk2pv7kg7vq1gjag3791hql2fg63d-openssl-1.0.2n-dev/include -I/nix/store/2hjyi8awwrmw5hk0py7j9iw9930l9bpy-openssl-1.0.2n/include -I/nix/store/3rs36zszvvwzckdr99vkbsmcydfj09xs-libpcap-1.8.1/include -fno-omit-frame-pointer -fno-strict-aliasing -ggdb -pthread -Wall -Wsign-compare -Wno-unknown-pragmas -Winvalid-pch -O2 -Wno-unused-local-typedefs -Wno-unused-function -Wno-deprecated-declarations -Wno-unused-const-variable -Wno-unused-but-set-variable -Wno-missing-braces -fstack-protector-strong -fno-builtin-memcmp -fPIE -DNDEBUG -D_FORTIFY_SOURCE=2 -DBOOST_THREAD_VERSION=4 -DBOOST_THREAD_DONT_PROVIDE_VARIADIC_THREAD -DBOOST_SYSTEM_NO_DEPRECATED -DBOOST_OPTIONAL_USE_SINGLETON_DEFINITION_OF_NONE -DHAVE_VERBOSE -DHAVE_FALLOCATE -DHAVE_SYNC_FILE_RANGE -D_GNU_SOURCE -DHAVE_BUILTIN_EXTENSION_ZLIB -DHAVE_BUILTIN_EXTENSION_SNAPPY -DHAVE_CRC32_HARDWARE -Ibuild/nixos/third_party/wiredtiger -Isrc/third_party/wiredtiger -Ibuild/nixos/third_party/wiredtiger/src/include -Isrc/third_party/wiredtiger/src/include -Ibuild/nixos/third_party/wiredtiger/build_linux -Isrc/third_party/wiredtiger/build_linux src/third_party/wiredtiger/src/cursor/cur_index.c
gcc -o build/nixos/third_party/wiredtiger/src/cursor/cur_join.o -c -std=c11 -I/nix/store/60q6d1a7s464izwbxq9gi0q5m3x6kjiw-cyrus-sasl-2.1.26-bin/include -I/nix/store/ybgzdhff90g8hrkv4c9863ynh9h4ldz6-boost-1.60.0/include -I/nix/store/5v9q166fyigywngdajcgh1wd8f0m62l5-gperftools-2.6.1/include -I/nix/store/hhz1a55j3mmwz6z4bc8crhm3qi5gm2l1-pcre-cpp-8.41-bin/include -I/nix/store/k2mcv7x6v4wrq9ldjzqa4by3g2fkk0zp-snappy-1.1.7/include -I/nix/store/wk9ngxwd8i3fiqhp4df201bqklfp2f4j-zlib-1.2.11/include -I/nix/store/rk8833kc9imj5mf4y1v3v6ykj4asfp18-libyaml-cpp-0.6.0/include -I/nix/store/60q6d1a7s464izwbxq9gi0q5m3x6kjiw-cyrus-sasl-2.1.26-bin/include -I/nix/store/sk5fk2pv7kg7vq1gjag3791hql2fg63d-openssl-1.0.2n-dev/include -I/nix/store/2hjyi8awwrmw5hk0py7j9iw9930l9bpy-openssl-1.0.2n/include -I/nix/store/3rs36zszvvwzckdr99vkbsmcydfj09xs-libpcap-1.8.1/include -fno-omit-frame-pointer -fno-strict-aliasing -ggdb -pthread -Wall -Wsign-compare -Wno-unknown-pragmas -Winvalid-pch -O2 -Wno-unused-local-typedefs -Wno-unused-function -Wno-deprecated-declarations -Wno-unused-const-variable -Wno-unused-but-set-variable -Wno-missing-braces -fstack-protector-strong -fno-builtin-memcmp -fPIE -DNDEBUG -D_FORTIFY_SOURCE=2 -DBOOST_THREAD_VERSION=4 -DBOOST_THREAD_DONT_PROVIDE_VARIADIC_THREAD -DBOOST_SYSTEM_NO_DEPRECATED -DBOOST_OPTIONAL_USE_SINGLETON_DEFINITION_OF_NONE -DHAVE_VERBOSE -DHAVE_FALLOCATE -DHAVE_SYNC_FILE_RANGE -D_GNU_SOURCE -DHAVE_BUILTIN_EXTENSION_ZLIB -DHAVE_BUILTIN_EXTENSION_SNAPPY -DHAVE_CRC32_HARDWARE -Ibuild/nixos/third_party/wiredtiger -Isrc/third_party/wiredtiger -Ibuild/nixos/third_party/wiredtiger/src/include -Isrc/third_party/wiredtiger/src/include -Ibuild/nixos/third_party/wiredtiger/build_linux -Isrc/third_party/wiredtiger/build_linux src/third_party/wiredtiger/src/cursor/cur_join.c
gcc -o build/nixos/third_party/wiredtiger/src/cursor/cur_json.o -c -std=c11 -I/nix/store/60q6d1a7s464izwbxq9gi0q5m3x6kjiw-cyrus-sasl-2.1.26-bin/include -I/nix/store/ybgzdhff90g8hrkv4c9863ynh9h4ldz6-boost-1.60.0/include -I/nix/store/5v9q166fyigywngdajcgh1wd8f0m62l5-gperftools-2.6.1/include -I/nix/store/hhz1a55j3mmwz6z4bc8crhm3qi5gm2l1-pcre-cpp-8.41-bin/include -I/nix/store/k2mcv7x6v4wrq9ldjzqa4by3g2fkk0zp-snappy-1.1.7/include -I/nix/store/wk9ngxwd8i3fiqhp4df201bqklfp2f4j-zlib-1.2.11/include -I/nix/store/rk8833kc9imj5mf4y1v3v6ykj4asfp18-libyaml-cpp-0.6.0/include -I/nix/store/60q6d1a7s464izwbxq9gi0q5m3x6kjiw-cyrus-sasl-2.1.26-bin/include -I/nix/store/sk5fk2pv7kg7vq1gjag3791hql2fg63d-openssl-1.0.2n-dev/include -I/nix/store/2hjyi8awwrmw5hk0py7j9iw9930l9bpy-openssl-1.0.2n/include -I/nix/store/3rs36zszvvwzckdr99vkbsmcydfj09xs-libpcap-1.8.1/include -fno-omit-frame-pointer -fno-strict-aliasing -ggdb -pthread -Wall -Wsign-compare -Wno-unknown-pragmas -Winvalid-pch -O2 -Wno-unused-local-typedefs -Wno-unused-function -Wno-deprecated-declarations -Wno-unused-const-variable -Wno-unused-but-set-variable -Wno-missing-braces -fstack-protector-strong -fno-builtin-memcmp -fPIE -DNDEBUG -D_FORTIFY_SOURCE=2 -DBOOST_THREAD_VERSION=4 -DBOOST_THREAD_DONT_PROVIDE_VARIADIC_THREAD -DBOOST_SYSTEM_NO_DEPRECATED -DBOOST_OPTIONAL_USE_SINGLETON_DEFINITION_OF_NONE -DHAVE_VERBOSE -DHAVE_FALLOCATE -DHAVE_SYNC_FILE_RANGE -D_GNU_SOURCE -DHAVE_BUILTIN_EXTENSION_ZLIB -DHAVE_BUILTIN_EXTENSION_SNAPPY -DHAVE_CRC32_HARDWARE -Ibuild/nixos/third_party/wiredtiger -Isrc/third_party/wiredtiger -Ibuild/nixos/third_party/wiredtiger/src/include -Isrc/third_party/wiredtiger/src/include -Ibuild/nixos/third_party/wiredtiger/build_linux -Isrc/third_party/wiredtiger/build_linux src/third_party/wiredtiger/src/cursor/cur_json.c
gcc -o build/nixos/third_party/wiredtiger/src/cursor/cur_log.o -c -std=c11 -I/nix/store/60q6d1a7s464izwbxq9gi0q5m3x6kjiw-cyrus-sasl-2.1.26-bin/include -I/nix/store/ybgzdhff90g8hrkv4c9863ynh9h4ldz6-boost-1.60.0/include -I/nix/store/5v9q166fyigywngdajcgh1wd8f0m62l5-gperftools-2.6.1/include -I/nix/store/hhz1a55j3mmwz6z4bc8crhm3qi5gm2l1-pcre-cpp-8.41-bin/include -I/nix/store/k2mcv7x6v4wrq9ldjzqa4by3g2fkk0zp-snappy-1.1.7/include -I/nix/store/wk9ngxwd8i3fiqhp4df201bqklfp2f4j-zlib-1.2.11/include -I/nix/store/rk8833kc9imj5mf4y1v3v6ykj4asfp18-libyaml-cpp-0.6.0/include -I/nix/store/60q6d1a7s464izwbxq9gi0q5m3x6kjiw-cyrus-sasl-2.1.26-bin/include -I/nix/store/sk5fk2pv7kg7vq1gjag3791hql2fg63d-openssl-1.0.2n-dev/include -I/nix/store/2hjyi8awwrmw5hk0py7j9iw9930l9bpy-openssl-1.0.2n/include -I/nix/store/3rs36zszvvwzckdr99vkbsmcydfj09xs-libpcap-1.8.1/include -fno-omit-frame-pointer -fno-strict-aliasing -ggdb -pthread -Wall -Wsign-compare -Wno-unknown-pragmas -Winvalid-pch -O2 -Wno-unused-local-typedefs -Wno-unused-function -Wno-deprecated-declarations -Wno-unused-const-variable -Wno-unused-but-set-variable -Wno-missing-braces -fstack-protector-strong -fno-builtin-memcmp -fPIE -DNDEBUG -D_FORTIFY_SOURCE=2 -DBOOST_THREAD_VERSION=4 -DBOOST_THREAD_DONT_PROVIDE_VARIADIC_THREAD -DBOOST_SYSTEM_NO_DEPRECATED -DBOOST_OPTIONAL_USE_SINGLETON_DEFINITION_OF_NONE -DHAVE_VERBOSE -DHAVE_FALLOCATE -DHAVE_SYNC_FILE_RANGE -D_GNU_SOURCE -DHAVE_BUILTIN_EXTENSION_ZLIB -DHAVE_BUILTIN_EXTENSION_SNAPPY -DHAVE_CRC32_HARDWARE -Ibuild/nixos/third_party/wiredtiger -Isrc/third_party/wiredtiger -Ibuild/nixos/third_party/wiredtiger/src/include -Isrc/third_party/wiredtiger/src/include -Ibuild/nixos/third_party/wiredtiger/build_linux -Isrc/third_party/wiredtiger/build_linux src/third_party/wiredtiger/src/cursor/cur_log.c
gcc -o build/nixos/third_party/wiredtiger/src/cursor/cur_metadata.o -c -std=c11 -I/nix/store/60q6d1a7s464izwbxq9gi0q5m3x6kjiw-cyrus-sasl-2.1.26-bin/include -I/nix/store/ybgzdhff90g8hrkv4c9863ynh9h4ldz6-boost-1.60.0/include -I/nix/store/5v9q166fyigywngdajcgh1wd8f0m62l5-gperftools-2.6.1/include -I/nix/store/hhz1a55j3mmwz6z4bc8crhm3qi5gm2l1-pcre-cpp-8.41-bin/include -I/nix/store/k2mcv7x6v4wrq9ldjzqa4by3g2fkk0zp-snappy-1.1.7/include -I/nix/store/wk9ngxwd8i3fiqhp4df201bqklfp2f4j-zlib-1.2.11/include -I/nix/store/rk8833kc9imj5mf4y1v3v6ykj4asfp18-libyaml-cpp-0.6.0/include -I/nix/store/60q6d1a7s464izwbxq9gi0q5m3x6kjiw-cyrus-sasl-2.1.26-bin/include -I/nix/store/sk5fk2pv7kg7vq1gjag3791hql2fg63d-openssl-1.0.2n-dev/include -I/nix/store/2hjyi8awwrmw5hk0py7j9iw9930l9bpy-openssl-1.0.2n/include -I/nix/store/3rs36zszvvwzckdr99vkbsmcydfj09xs-libpcap-1.8.1/include -fno-omit-frame-pointer -fno-strict-aliasing -ggdb -pthread -Wall -Wsign-compare -Wno-unknown-pragmas -Winvalid-pch -O2 -Wno-unused-local-typedefs -Wno-unused-function -Wno-deprecated-declarations -Wno-unused-const-variable -Wno-unused-but-set-variable -Wno-missing-braces -fstack-protector-strong -fno-builtin-memcmp -fPIE -DNDEBUG -D_FORTIFY_SOURCE=2 -DBOOST_THREAD_VERSION=4 -DBOOST_THREAD_DONT_PROVIDE_VARIADIC_THREAD -DBOOST_SYSTEM_NO_DEPRECATED -DBOOST_OPTIONAL_USE_SINGLETON_DEFINITION_OF_NONE -DHAVE_VERBOSE -DHAVE_FALLOCATE -DHAVE_SYNC_FILE_RANGE -D_GNU_SOURCE -DHAVE_BUILTIN_EXTENSION_ZLIB -DHAVE_BUILTIN_EXTENSION_SNAPPY -DHAVE_CRC32_HARDWARE -Ibuild/nixos/third_party/wiredtiger -Isrc/third_party/wiredtiger -Ibuild/nixos/third_party/wiredtiger/src/include -Isrc/third_party/wiredtiger/src/include -Ibuild/nixos/third_party/wiredtiger/build_linux -Isrc/third_party/wiredtiger/build_linux src/third_party/wiredtiger/src/cursor/cur_metadata.c
scons: *** [build/nixos/third_party/wiredtiger/src/checksum/arm64/crc32-arm64.o] Error 1
scons: building terminated because of errors.
build/nixos/third_party/wiredtiger/src/checksum/arm64/crc32-arm64.o failed: Error 1
builder for '/nix/store/chbzgmgf47assz9h0rllhg97268ad571-mongodb-3.4.10.drv' failed with exit code 2
error: build of '/nix/store/5lrf65gyzp3mbxpxmay50dxzz6hravp5-powerdns-4.0.5.drv', '/nix/store/chbzgmgf47assz9h0rllhg97268ad571-mongodb-3.4.10.drv' failed
```
</p></details>

GrahamcOfBorg reviewed (commented):
Failure on x86_64-linux [(full log)](https://logs.nix.gsc.io/?key=nixos/nixpkgs.34426)

<details><summary>Partial log (click to expand)</summary><p>

```
libtool: link: cannot determine absolute directory name of `lib'
make[3]: *** [Makefile:544: libgeoipbackend.la] Error 1
make[3]: Leaving directory '/tmp/nix-build-powerdns-4.0.5.drv-0/pdns-4.0.5/modules/geoipbackend'
make[2]: *** [Makefile:491: all-recursive] Error 1
make[2]: Leaving directory '/tmp/nix-build-powerdns-4.0.5.drv-0/pdns-4.0.5/modules'
make[1]: *** [Makefile:539: all-recursive] Error 1
make[1]: Leaving directory '/tmp/nix-build-powerdns-4.0.5.drv-0/pdns-4.0.5'
make: *** [Makefile:470: all] Error 2
builder for ‘/nix/store/y9c1q98c0rxmzkzz30888a4yhwb2pq46-powerdns-4.0.5.drv’ failed with exit code 2
error: build of ‘/nix/store/y9c1q98c0rxmzkzz30888a4yhwb2pq46-powerdns-4.0.5.drv’, ‘/nix/store/zqnfbclcg58dr3lpn86q5wi3apcanhgg-mongodb-3.4.10.drv’ failed
```
</p></details>

andir commented:
(Writing my WIP notes here so I can open issues later on or work on those directly)

Ok, there seemse to be (not just locally) some weird build errors with `powerdns`, `openxcom` and `ring-daemon`.

- `ring-daemon` seems to fail since a while. That is most likely due to `pjsip` being a newer version now. The last successful build was somewhere last year: https://hydra.nixos.org/build/57378758

- `powerdns` fails with the following log, reverting 133ff57 fixes it...
  ```
  /nix/store/i0ay05pqkbnvpfijm52mmlrp6kmkl80c-bash-4.4-p12/bin/bash ../../libtool  --tag=CXX   --mode=link g++  -fPIE -DPIE -U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=2 --param ssp-buffer-size=4 -fstack-protector -Wall -g -O2 -module -avoid-version -Wl,-z -Wl,relro -Wl,-z -Wl,now  -o libgeoipbackend.la -rpath /nix/store/45xa0g9x67x4rg4j25d48ipcwrh2ssdz-powerdns-4.0.5/lib/pdns geoipbackend.lo -Llib -lyaml-cpp -L/nix/store/f600fiy35cvzdaq91x6hhazpkh6w679d-geoip-1.6.12/lib -lGeoIP 
  ../../libtool: line 6000: cd: lib: No such file or directory
  libtool: link: cannot determine absolute directory name of `lib'
  ```

- `openxcom` fails since `ceil` is now suddenly undeclared despite `<cmath>` being imported. Failing lines are https://github.com/SupSuper/OpenXcom/blob/master/src/Basescape/CraftInfoState.cpp#L168

andir commented:
@GrahamcOfBorg build powerdns

GrahamcOfBorg commented:
Failure on x86_64-darwin [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34426&attempt_id=8f33e458-9de9-4a30-b5ec-756c0177963d)

<details><summary>Partial log (click to expand)</summary><p>

```
Package ‘powerdns-4.0.5’ in /private/var/lib/ofborg/checkout/repo/38dca4e3aa6bca43ea96d2fcc04e8229/builder/lnl7-mac/pkgs/servers/dns/powerdns/default.nix:35 is not supported on ‘x86_64-darwin’, refusing to evaluate.

a) For `nixos-rebuild` you can set
  { nixpkgs.config.allowBroken = true; }
in configuration.nix to override this.

b) For `nix-env`, `nix-build`, `nix-shell` or any other Nix command you can add
  { allowBroken = true; }
to ~/.config/nixpkgs/config.nix.

```
</p></details>

andir committed:
openxcom: 1.0.0 -> 1.0.0.2018.01.28

Update to a recent git head since porting more compiler fixes without
affecting the gameplay/features would require writing custom patches.

The current maintainer (@cpages) ack'ed this change via IRC.

GrahamcOfBorg commented:
Success on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34426&attempt_id=4751ff93-d933-4ee9-9a91-2eb0af5862e2)

<details><summary>Partial log (click to expand)</summary><p>

```
shrinking /nix/store/jcbkcmycxzkv5l3xi4x375vbgibdlry8-powerdns-4.0.5/lib/pdns/libremotebackend.so
shrinking /nix/store/jcbkcmycxzkv5l3xi4x375vbgibdlry8-powerdns-4.0.5/lib/pdns/librandombackend.so
shrinking /nix/store/jcbkcmycxzkv5l3xi4x375vbgibdlry8-powerdns-4.0.5/lib/pdns/libgsqlite3backend.so
gzipping man pages under /nix/store/jcbkcmycxzkv5l3xi4x375vbgibdlry8-powerdns-4.0.5/share/man/
strip is /nix/store/5qj61lcvzlap87rf6blvf8p577d482bv-binutils-2.28.1/bin/strip
stripping (with command strip and flags -S) in /nix/store/jcbkcmycxzkv5l3xi4x375vbgibdlry8-powerdns-4.0.5/lib  /nix/store/jcbkcmycxzkv5l3xi4x375vbgibdlry8-powerdns-4.0.5/bin  /nix/store/jcbkcmycxzkv5l3xi4x375vbgibdlry8-powerdns-4.0.5/sbin 
patching script interpreter paths in /nix/store/jcbkcmycxzkv5l3xi4x375vbgibdlry8-powerdns-4.0.5
checking for references to /tmp/nix-build-powerdns-4.0.5.drv-0 in /nix/store/jcbkcmycxzkv5l3xi4x375vbgibdlry8-powerdns-4.0.5...
moving /nix/store/jcbkcmycxzkv5l3xi4x375vbgibdlry8-powerdns-4.0.5/sbin/* to /nix/store/jcbkcmycxzkv5l3xi4x375vbgibdlry8-powerdns-4.0.5/bin
/nix/store/jcbkcmycxzkv5l3xi4x375vbgibdlry8-powerdns-4.0.5
```
</p></details>

GrahamcOfBorg commented:
Success on aarch64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.34426&attempt_id=bab2acfd-00fd-443b-9bfd-f2433be27e07)

<details><summary>Partial log (click to expand)</summary><p>

```
shrinking /nix/store/pwvh5f5jimd55b5wkfqf728n81iv8df7-powerdns-4.0.5/lib/pdns/libgeoipbackend.so
shrinking /nix/store/pwvh5f5jimd55b5wkfqf728n81iv8df7-powerdns-4.0.5/lib/pdns/libgmysqlbackend.so
shrinking /nix/store/pwvh5f5jimd55b5wkfqf728n81iv8df7-powerdns-4.0.5/lib/pdns/libbindbackend.so
gzipping man pages under /nix/store/pwvh5f5jimd55b5wkfqf728n81iv8df7-powerdns-4.0.5/share/man/
strip is /nix/store/xmpjypwjmp2qi1chs5kr0hacnh161ls4-binutils-2.28.1/bin/strip
stripping (with command strip and flags -S) in /nix/store/pwvh5f5jimd55b5wkfqf728n81iv8df7-powerdns-4.0.5/lib  /nix/store/pwvh5f5jimd55b5wkfqf728n81iv8df7-powerdns-4.0.5/bin  /nix/store/pwvh5f5jimd55b5wkfqf728n81iv8df7-powerdns-4.0.5/sbin
patching script interpreter paths in /nix/store/pwvh5f5jimd55b5wkfqf728n81iv8df7-powerdns-4.0.5
checking for references to /build in /nix/store/pwvh5f5jimd55b5wkfqf728n81iv8df7-powerdns-4.0.5...
moving /nix/store/pwvh5f5jimd55b5wkfqf728n81iv8df7-powerdns-4.0.5/sbin/* to /nix/store/pwvh5f5jimd55b5wkfqf728n81iv8df7-powerdns-4.0.5/bin
/nix/store/pwvh5f5jimd55b5wkfqf728n81iv8df7-powerdns-4.0.5
```
</p></details>

andir committed:
ring-daemon: mark as broken since the patched pjsip fails to build

The patches do no longer apply on-top of the pjsip version we provice.
One of the maintainers (@Radvendii @olynch) should have a look what can
be done about this.

The build has been failing since 2017-07-31…
